campaign/record_unit.xml

```diff
@@ -475,4 +482,91 @@
 			</basicstring>
 		</sheetdata>
 	</windowclass>
+
+	<windowclass name="unit_effects">
+		<margins control="0,0,0,2" />
+		<script>
+			function onInit()    
+				update();
+			end
+
+			function onClose()
```

Comment:
Any plans to add something here?

Reply:
Relic from when I was using handlers for something.